modules/AOW_WorkFlow/AOW_WorkFlow.php

```diff
@@ -370,6 +383,18 @@ public function build_query_where(AOW_Condition $condition, $module, $query = ar
                 return $query;
             }
 
+            if ($condition->isDateAnniversaryCondition()) {
+                $this->isAnniversaryWf = true;
+                $where = '('.$field.' '.$this->getSQLOperator('Anniversary');
```

Comment:
`$field` value should be escaped before using it in an sql query. You could use the DBManager::quote and quoted methods.